app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -426,6 +426,9 @@ public void onResume() {
         for (WindowWidget window: mPrivateWindows) {
             window.onResume();
         }
+
+        TelemetryWrapper.resetOpenedWindowsCount(mRegularWindows.size(), false);
+        TelemetryWrapper.resetOpenedWindowsCount(mPrivateWindows.size(), true);
```

Comment:
@keianhzo  Here still looks odd to me.
Like you said, `when openWindows has three buckets so when setting to one we are basically initializing the bucket for that windows number.`
If so, for example, when mRegularWindows(2) to do initialize, you need to make
openWindows[0] = 1;
openWindows[1] = 1;
But, you only make openWindows[2 -1] = 1;